test/unittests/common/commonTag_test.cpp

```diff
@@ -120,7 +120,8 @@ TEST(commonTag, valueTag)
    out = valueTag(indent, tag, value, JSON);
    EXPECT_EQ(jsonNoComma, out);   
 
-   out = valueTag(indent, tag, value, JSON, true, true);
+   // FIXME: after removing association code, maybe the test for 'jsonCommaAndAssociation' is no longer needed
```

Comment:
From the text 'jsonCommaAndAssociation' it **really** seems it should be removed.